packages/frontend/components/Plan/AddPlanModal.tsx

```diff
@@ -243,16 +260,24 @@ export const AddPlanModal: React.FC<AddPlanModalProps> = ({
                         setValue("concentration", "");
                       }}
                       rules={{ required: "Major is required." }}
-                      helperText='First select your catalog year. If you still cannot find your major, select "No Major" above.'
+                      isDisabled={!!!catalogYear}
                       isSearchable
                       useFuzzySearch
                     />
-                    <PlanConcentrationsSelect
-                      catalogYear={catalogYear}
-                      majorName={majorName}
-                      supportedMajorsData={supportedMajorsData}
-                      control={control}
-                    />
+                    {majorHasConcentrations && (
+                      <PlanSelect
+                        label="Concentrations"
+                        noValueOptionLabel="Select a Concentration"
+                        name="concentration"
+                        options={selectedMajorConcentrations.concentrations}
+                        control={control}
+                        rules={{
+                          required:
+                            isConcentrationRequired &&
+                            "Concentration is required",
+                        }}
+                      />
+                    )}
```

Comment:
I moved everything from the `PlanConcentrationSelect` component since we need the isConcentrationRequired logic for `isValidForm` and that was basically all the logic from that component.

@BrandonLim8890 should we delete `PlanConcentrationSelect`?